packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -169,10 +173,10 @@ export const AuthModal = ({
                 try {
                     await getPasswordResetToken(data.email)
                 } catch (e) {
-                    form.setError('global', {
-                        type: 'manual',
-                        message: formatMessage(API_ERROR_MESSAGE)
-                    })
+                    const message = /^400/.test(e.message)
```

Comment:
instead of checking the string would we consider checking the status code directly to be more reliable in case the error message changes where `400` is not the first part of the string